[UIK-2637][feedback-form] Update feedback rating a11y (#1815)
semrush/intergalactic: semcore/feedback-form/CHANGELOG.md

CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).

## [6.38.0] - 2024-12-06

### Added

> - `aria-describedby`, that refers to ScreenRiderOnly text, `aria-labelledby` referring to the notice text, `aria-valuetext="{x} out of {max}"` to SliderRating in FeedbackRating.

Reviewer: ```suggestion
- `aria-describedby`, that refers to ScreenRiderOnly text, `aria-labelledby` referring to the notice text, `aria-valuetext` to SliderRating.
```

> - `role="image"` and `aria-label="Your rating: {x} out of 5"` for SliderRating's readonly mode in FeedbackRating.

Reviewer: ```suggestion
- `role="image"` and `aria-label` for SliderRating's readonly mode.
```

> - `aria-labelledby=<Header>` to FeedbackRating modal.

Reviewer: ```suggestion
- `aria-labelledby` for the FeedbackRating's modal container, that refers to Header of it.
```

- autofocus to first checkbox. Moved from `textarea`.

> - added `aria-describedby`, that connects privacy-description text and related form control.

Reviewer: ```suggestion
- `aria-describedby`, that connects privacy-description text and related form control in FeedbackRating form.
```

### Changed

> - color of privacy-description text `text-secondary` token.

Reviewer: ```suggestion
- color of privacy-description text to the `text-secondary` token in FeedbackRating form.
```

- `FeedbackRating.Header` as optional, when specifying FeedbackRating's `header` prop.

### Removed

> - `title` attribute from from FeedbackRating form.

Reviewer: ```suggestion
- `title` attribute from FeedbackRating form.
```

> - removed `<ul><li>` structure from checkboxes, leaving only `div role="group"`.

Reviewer: ```suggestion
- removed `<ul><li>` structure from checkboxes, leaving only `div role="group"` in FeedbackRating form.
```

- removed unnecessary tooltip around `input[type="hidden"]`

> - removed `line-height` from privacy-description text.

Reviewer: ```suggestion
- removed `line-height` from privacy-description text in FeedbackRating form.
```

CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).

> ## [6.37.1] - 2024-12-05

Reviewer: I think you should stay it as is and just add 6.38.0 for you a11y changes.
Also maybe let's write some points like `added aria-haspopup for controls with errors`, `added dynamic role slider or img depends on readonly property for Slider` and so on

Author: Added

CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).

## [6.37.1] - 2024-11-29

### Changed

- FeedbackRating form a11y

> ## [6.37.0] - 2024-12-02

Reviewer: strange order of dates

Author: Fixed
